<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SPasClass</class>
 <widget class="QMainWindow" name="SPasClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SPas</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhHiddenText</set>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout" rowstretch="6,0,3" columnstretch="1,0" rowminimumheight="0,0,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QTableWidget" name="FrameListTableWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="columnCount">
          <number>11</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>帧序号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>帧类型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>帧长度</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>帧偏移</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>音视频类型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>编码类型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>编码格式</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>分辨率/宽</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>分辨率/高</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>帧率</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>时间戳</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="FrameShowBinaryTextLabel">
         <property name="text">
          <string>Binary Text</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="MediaInfoLable">
         <property name="text">
          <string>Media Info</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QTextEdit" name="FrameShowBinaryText"/>
       </item>
       <item row="2" column="1">
        <widget class="QTextEdit" name="MediaInfo"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>964</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="openFile"/>
   </widget>
   <widget class="QMenu" name="menuview">
    <property name="title">
     <string>view</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuinfo">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuview"/>
   <addaction name="menuinfo"/>
  </widget>
  <action name="openFile">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="videoSpas.qrc"/>
 </resources>
 <connections/>
</ui>
